src/Tribe/Tickets.php

```diff
@@ -678,7 +679,7 @@ public function has_permission( $post, $data, $nonce_action ) {
 			$ticket->ID             = isset( $data['ticket_id'] ) ? absint( $data['ticket_id'] ) : null;
 			$ticket->name           = isset( $data['ticket_name'] ) ? esc_html( $data['ticket_name'] ) : null;
 			$ticket->description    = isset( $data['ticket_description'] ) ? esc_html( $data['ticket_description'] ) : null;
-			$ticket->price          = ! empty( $data['ticket_price'] ) ? trim( $data['ticket_price'] ) : 0;
+			$ticket->price          = ! empty( $data['ticket_price'] ) ? absint( trim( $data['ticket_price'] ) ) : 0;
```

Comment:
Hmm ... well, not perfect by a hundred miles, but something along the lines of the following would at least filter out non-numerals/non-separators even if it doesn't cover all possible scenarios:

`filter_var( $value, FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION | FILTER_FLAG_ALLOW_THOUSAND );`

... I daresay WooCommerce and co have this problem licked - so perhaps we could also draw inspiration and create an appropriate helper function in `tribe-common` (though I'm personally happy with whatever works well in most predictable cases, so long as it's secure).